Vic2ToHoI4/Data_Files/blankMod/output/localisation/events2_l_english.yml

```diff
@@ -513,3 +513,15 @@
  olympics.failed.t:10 "The Cancellation of the Olympic Games"
  olympics.failed.d:10 "Due to the current international situation, the Olympic Games scheduled for [GetDateStringNoHour] could not be held."
  olympics.failed.a:10 "Not my fault."
+ neighbor_war.1.t:0 "[?neighbor_war_attacker.GetName] wins"
+ neighbor_war.1.d:0 "[?neighbor_war_attacker.GetName] was victorious in the [?neighbor_war_attacker.GetAdjectiveCap]-[?neighbor_war_defender.GetAdjectiveCap] conflict."
+ neighbor_war.1.a:0 "Glory to [?neighbor_war_attacker.GetName]"
+ neighbor_war.1.b:0 "Bleeding [?neighbor_war_attacker.GetAdjectiveCap]s"
+ neighbor_war.2.t:0 "[?neighbor_war_defender.GetName] wins"
+ neighbor_war.2.d:0 "[?neighbor_war_defender.GetName] was victorious in the [?neighbor_war_attacker.GetAdjectiveCap]-[?neighbor_war_defender.GetAdjectiveCap] conflict."
+ neighbor_war.2.a:0 "Glory to [?neighbor_war_defender.GetName]"
+ neighbor_war.2.b:0 "We will have our revenge"
+ neighbor_war.3.t:0 "[?neighbor_war_attacker.GetAdjectiveCap]-[?neighbor_war_defender.GetAdjectiveCap] stalemate"
+ neighbor_war.3.d:0 "Neither side has been able to secure the win in the [?neighbor_war_attacker.GetAdjectiveCap]-[?neighbor_war_defender.GetAdjectiveCap] conflict."
+ neighbor_war.3.a:0 "We'll try again"
+ neighbor_war.3.b:0 "Thanks to our brave soldier"
```

Comment:
Ask the friendly folks on Discord to provide non-english localisations for these.